packages/babel-plugin-transform-flow-comments/src/index.js

```diff
@@ -37,6 +37,12 @@ export default function ({ types: t }) {
         }
       },
 
+      // support for `class X { foo: string }` - #4622
+      ClassProperty(path) {
+        let { node, parent } = path;
```

Comment:
this could be move to the declaration directly `ClassProperty({ node, parent })`, although I see we don't do it in the other places too.